src/rt/dmain2.d

```diff
@@ -477,6 +474,13 @@ extern (C) int _d_run_main(int argc, char **argv, MainFunc mainFunc)
             result = (result == EXIT_SUCCESS) ? EXIT_FAILURE : result;
     }
 
+    import rt.config : rt_configOption;
+
+    if (rt_configOption("noTrapExceptions") !is null)
```

Comment:
It looks like you're using the result of `rt_configOption` like a bool here... which means that `--DRT-noTrapExceptions=1` and `--DRT-noTrapExceptions=0` will both do the same thing. This is not ideal.

I looked to see if we have a precedent for this, and here is how #1798 added `callStructDtorsDuringGC`, also a boolean option: https://github.com/dlang/druntime/blob/639878dc9c02dfff9603f024988ccd4d5ecee2cb/src/rt/lifetime.d#L49-L58

I suggest to parse this option in a similar way, which would also allow e.g. overriding the default behaviour of disabling `rt_trapExceptions` when running under a debugger on Windows. That also solves the "double negation" issue. If you're feeling up for it, refactoring the bool checks into a new function in the `rt.config` module, which also rejects invalid values, would be a bonus.